server/models/session.js

```diff
@@ -33,7 +33,7 @@ class Session extends MongoModels {
         });
     }
 
-    static create(userId, callback) {
+    static create(userId, ip, browser, os, callback) {
```

Comment:
Instead of passing `browser` and `os`, maybe we could pass the useragent string all the way down to where it's used and then use the `useragent` module to parse it there... or we could send down the parsed useragent object and use the parts of it we're interested in.